napari/components/cursor.py

```diff
@@ -36,6 +36,6 @@ class Cursor(EventedModel):
     # fields
     position: Tuple[float, ...] = (1, 1)
     scaled: bool = True
-    size: int = 1
+    size: float = 1
```

Comment:
```suggestion
    size: float = 1.0
```

?

Reply:
I'll change this to `size = 1.0` to make it clearer this is a float without needing an explicit annotation.